lib/internal/timers.js

```diff
@@ -627,17 +632,22 @@ class Immediate {
   ref() {
     if (this[kRefed] === false) {
       this[kRefed] = true;
-      if (immediateInfo[kRefCount]++ === 0)
-        toggleImmediateRef(true);
+
+      if (immediateInfo[kRefCount]++ === 0) {
+       // We need to use the binding as the receiver for fast API calls.
```

Comment:
```suggestion
        // We need to use the binding as the receiver for fast API calls.
```
To make the linters happy.